packages/backend/src/tokens/request.ts

```diff
@@ -128,11 +136,11 @@ export type AuthenticateRequestOptions = InstanceKeys &
     /**
      * @experimental
      */
-    searchParams?: URLSearchParams;
+    signInUrl?: string;
     /**
      * @experimental
      */
-    signInUrl?: string;
+    isApiRoute?: boolean;
```

Comment:
@dimkl I think that we should make this more generic and call it `ignoreInterstitial` which is either a boolean or a function. We might more use cases for this such as assets, etc...

Reply:
I didn't add the function in type since the `url` of the request is not available in the `@clerk/backend` package.
We can do that after the addition of the `request` parameter (see: https://github.com/clerkinc/javascript/pull/1393)

Reply:
Correct. @nikosdouvlis how far are we from merging #1393 ?

Reply:
@SokratisVidros, i think we can continue with the `boolean` type and enhance it ( `boolean | ((url: URL | string) => boolean);`) when `request` parameter PR is merged.

Reply:
The `authenticateRequest` PR should be merged today, but lets do what @dimkl suggested.

Dimitri, can we add a JSDOC comment documenting what the prop does?

I'm not a fan of the `ignoreIInterstitial` name as it feels like an escape hatch and nothing a non-advanced Clerk user would understand. That being said, I don't have any better alternatives at the moment.